source/class/qx/tool/cli/Cli.js

```diff
@@ -283,8 +288,8 @@ Version: v${await qx.tool.config.Utils.getQxVersion()}
       var args = qx.lang.Array.clone(process.argv);
       args.shift();
       process.title = args.join(" ");
-      this.__promiseParseArgs = this.__parseArgsImpl();
-      await this.__promiseParseArgs;
+      await this.__parseArgsImpl();
```

Comment:
`__promiseParsedArgs` was used to avoid a race condition, I can't remember exactly why it came about, but it I expect it would be because something in `__parseArgsImpl` can refer recursively back to the `Cli.getParsedArgs()`.  Recursion will fail now, which is fine if that's been eliminated.   

Possibly add a check that `__parsedArgs` is non-null in `getParsedArgs`?

Reply:
Only usage is in Lint. There null value is handled